### ISSUE TITLE
Problem installing package

### ISSUE THREAD

**Issue by tcoussens** [rawlik/Coils.jl#1]
My appologies if this is a really simple solution. I'm not very familiar with the Julia language and I am having an issue installing the Coils package. I am using Julia 0.7 in JunoLab on a windows machine.

To install I have used the Pkg.clone("https://github.com/rawlik/Coils.jl.git") and throws the following error.
![image](https://user-images.githubusercontent.com/44604816/47719385-f1f33400-dc42-11e8-92ad-2b949e63e659.png)

This is the folder in question:
![image](https://user-images.githubusercontent.com/44604816/47720526-fc62fd00-dc45-11e8-8f66-a9c30e31fcff.png)

When testing to see if it installed using Pkg.status():
Note: I installed the dependencies manually.
![image](https://user-images.githubusercontent.com/44604816/47719469-354da280-dc43-11e8-8dee-62aae1f2a69f.png)

Then if I use instantiate command:
![image](https://user-images.githubusercontent.com/44604816/47719551-72199980-dc43-11e8-9876-6f8d53c00600.png)

I then if I try using the Coils package:
![image](https://user-images.githubusercontent.com/44604816/47719917-7db99000-dc44-11e8-8907-3314d21360bf.png)

Any help would be greatly appreciated. 

Thank you!

**Comment by mrawlik** [comment]
Hi,

This code is not a julia package. You don't install it with the julia package manager.

I guess by running Pkg.clone(...) you made julia package manager thing that there should be a Coils package, so it looks for one instead of loading the script in the folder (which is last in `LOAD_PATH`).

I guess if you run Pkg.rm("Coils") it is going to be back to normal.

The standard way to get the code would be either download it directly: https://github.com/rawlik/Coils.jl/archive/master.zip or clone the repository:
`$ git clone git@github.com:rawlik/Coils.jl.git`
(on windows you need git bash)
Both options are accessible via the green "clone or download" button.

**Comment by tcoussens** [comment]
Thank you for your reply. That explains a lot! Unfortunately I am still having a issue getting it to run.

By following the first steps in the example.ipynb file: 
![image](https://user-images.githubusercontent.com/44604816/47912089-54407480-de8f-11e8-8a79-fe3aaebf3836.png)

Then I run:
g, vertex_positions = cuboid_system([1, 1, 1], [3, 3, 3])
figure(figsize = (6, 6))

Then when I get to: plot_vertices(vertex_positions), Julia crashes.
![image](https://user-images.githubusercontent.com/44604816/47912279-f95b4d00-de8f-11e8-9de7-747793c5c1a1.png)

Any ideas?

Thanks Again!

**Comment by mrawlik** [comment]
Hi,

Yes, I noticed this problem, too. There were still some issues

I fixed them in this branch:
https://github.com/rawlik/Coils.jl/tree/juliapackage

You can install it as a normal julia package with:
```julia
(v1.0) pkg> add https://github.com/rawlik/Coils.jl #juliapackage
```

I don't merge it to master yet, as I want to fix the automatic travis tests first.

I hope this helps.

**Comment by mrawlik** [comment]
I merged now the changes into master (#2).  It runs with julia 1.0 and 0.7. I dropped compatibility with 0.6 altogether, as it was getting messy.

Now installation can be done by simply doing
```julia
(v1.0) pkg> add https://github.com/rawlik/Coils.jl
```

**Comment by tcoussens** [comment]
That's brilliant. All working now.

Thanks for your help!
